docs/index.md

```diff
@@ -35,19 +35,43 @@ brew tap drud/ddev && brew install ddev
 
 If you would like more frequent "edge" releases then use `brew tap drud/ddev-edge` instead.
 
-(Optional) As a one-time initialization, run `mkcert -install`. Linux users may have to take additional actions as discussed below in "Linux `mkcert -install` additional instructions".
+As a one-time initialization, run `mkcert -install`. Linux users may have to take additional actions as discussed below in "Linux `mkcert -install` additional instructions".
 
 Later, to upgrade to a newer version of ddev, run:
 
 ```
 ddev poweroff && brew upgrade ddev
 ```
 
-### Installation or Upgrade - Windows
+### Installation or Upgrade - Windows (WSL2)
+
+**This is the recommended installation method for all Windows users that are on Windows 10 2004 or higher.** If you don't have this version yet, or if you don't want to use WSL2, please follow the legacy instructions for Windows below.
+
+**All Windows 10 editions (including Windows 10 Home) support WSL2**. Docker Toolbox support for DDEV is deprecated and will be removed, as we'll move testing capacity towards WSL2. If you're already familiar with DDEV on Windows, you might have been using NFS for better filesystem performance. **You won't need NFS anymore once you switch to WSL2**, since it provides awesome filesystem performance out of the box 🚀
+
+Take the steps below to install DDEV into WSL2.
+
+* Install WSL2 by [following Microsoft's docs](https://docs.microsoft.com/en-us/windows/wsl/install-win10). Make sure the WSL version is set to 2: `wsl --set-default-version 2`
+* Install a WSL2 distro, e.g. [Ubuntu 20.04](https://www.microsoft.com/store/productId/9N6SVWS3RX71). Install Ubuntu (or any other distro) _before_ installing Docker, so that Ubuntu becomes the default WSL distro.
+* Install Docker Desktop for Windows (versions 2.3.0.2 and higher automatically use the WSL2 backend): [download Docker Desktop for Windows from Docker Hub](https://hub.docker.com/editions/community/docker-ce-desktop-windows/).
+* Go to Docker Desktop settings > Resources > WSL integration > enable integration for your distro (now `docker` commands will be available from within your WSL2 distro).
+* Install [Chocolatey](https://chocolatey.org/install) on Windows.
+* Open a git-bash terminal with administrator rights and install mkcert by running `choco install mkcert`
+* In git-bash, run `mkcert -install` and answer the prompt allowing the installation of the Certificate Authority.
+* In git-bash, run the command `setx CAROOT "$(mkcert -CAROOT)" && setx WSLENV "CAROOT/p;$WSLENV"`. This will set WSL2 to use the Certificate Authority installed on the Windows side.
+* Open the Ubuntu 20.04 terminal from the Windows start menu.
+* Run `export CAROOT=/mnt/c/Users/YOUR_WINDOWS_USERNAME/AppData/Local/mkcert`. This will make sure that `mkcert` on Linux uses your Windows certificates when issuing SSL certificates, so you don't get SSL warnings in your browser on Windows.
```

Comment:
If you set the variable on the Windows side and expose it to WSL like done above this is not needed anymore

Reply:
Thought I had already deleted that line, thanks.